data/s4shantanu.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Shantanu Raut",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Graduate || Backend Developer || Python || Java | |DSA || Leetcode 2🌟 || Hackerrank 5🌟 || Open-source || Postman Student Expert || Technical Volunteer in All About Programming || GDSC Member",
+  "avatar": "https://github.com/s4shantanu.png"
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/s4shaantanu",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/s4shantanu/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "Follow me on Linkedin",
+      "url": "https://www.linkedin.com/in/shantanu-raut/",
+      "icon": "FaLinkedin"
+    }
```

Add comma after curly braces and looks like this ` } ,`
```suggestion
    } ,
```